### ISSUE TITLE
backslashes move highlight position

### ISSUE THREAD

**Issue by 8633brown** [paulrosen/abcjs#436]
[typesetting_line-breaks](http://abcnotation.com/wiki/abc:standard:v2.1#typesetting_line-breaks)

backslashes can be used to ignore line-breaks. 
however when selecting a rendered note backslashes cause the text being highlighted to be push back by the number of backslashes before the selected text. 

for example...
```
cdef|
cedf:|
```
will select okay...
(as a more extreme example.)
```
cdef|
\
\
\
cedf:|
```
when selecting a note on the second line of the rendered music, the correct string will not be highlighted and instead the backslashes will be selected instead.

**Comment by 8633brown** [comment]
at the moment i've added 


```javascript
EditArea.prototype.getSelection = function() {
  const blankLines = (this.textarea.value.substring(0, this.textarea.selectionEnd).split('\n\\\n').length - 1) * 2
  return {start: this.textarea.selectionStart - blankLines, end: this.textarea.selectionEnd - blankLines};
};
```
[here](https://github.com/paulrosen/abcjs/blob/70faf7ef0a0742733ca14da3266b35bdfa6b7065/src/edit/abc_editor.js#L102-L104)

and

```javascript
EditArea.prototype.setSelection = function(start, end) {
	const blankLines = (this.textarea.value.substring(0, end).split('\n\\\n').length - 1) * 2;
	start += blankLines;
	end += blankLines;
	if(this.textarea.setSelectionRange)
	   this.textarea.setSelectionRange(start, end);
	else if(this.textarea.createTextRange) {
		// For IE8
	   var e = this.textarea.createTextRange();
	   e.collapse(true);
	   e.moveEnd('character', end);
	   e.moveStart('character', start);
	   e.select();
	}
  this.textarea.focus();
};
```
[here](https://github.com/paulrosen/abcjs/blob/70faf7ef0a0742733ca14da3266b35bdfa6b7065/src/edit/abc_editor.js#L106-L118)

also https://github.com/paulrosen/abcjs/blob/70faf7ef0a0742733ca14da3266b35bdfa6b7065/src/edit/abc_editor.js#L107 never evaluates to true so the fix for IE is the only part used

**Comment by paulrosen** [comment]
fixed in 6.0.0-beta.11
